spec.bs

```diff
@@ -2939,6 +2939,15 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=list=] of [=bid debug report
     [=interest group/owner=] is |igPair|'s [=interest group/owner=] and [=interest group/name=] is |igPair|'s
     [=interest group/name=]. [=iteration/Continue=] if none found.
   1. [=list/Append=] |ig| to |bidIgs|.
+1. [=map/For each=] |igPair| → |updateIfOlderThan| of |response|'s
+   [=server auction response/update groups=] field:
+  1. Let |ig| be the [=interest group=] in the [=user agent=]'s [=interest group set=] whose
+    [=interest group/owner=] is |igPair|'s [=interest group/owner=] and [=interest group/name=] is |igPair|'s
+    [=interest group/name=]. [=iteration/Continue=] if none found.
+  1. If |updateIfOlderThan| is less than 10 mintues, set it to 10 minutes.
+  1. If [=current wall time=] &minus; |ig|'s [=interest group/last updated=] ≥
+     |updateIfOlderThan|, set |ig|'s [=interest group/next update after=] to the

```

Comment:
not directly to this pull request but related, shoud [this](https://github.com/WICG/turtledove/blob/main/spec.bs#L5441) also be set to `[=current wall time=] + |updateIfOlderThan|` instead of `[=current wall time=]`, and also `set |ig|'s last updated to current wall time`?

Reply:
I changed it to `set |ig|'s [=interest group/next update after=]`, but I don't think we should set `[=interest group/last updated=]` here -- that happens when we actually perform an update.